pact/http/consumer-one/build.gradle.kts

```diff
@@ -15,3 +16,14 @@ dependencies {
     testImplementation("org.springframework.boot:spring-boot-starter-test")
     testImplementation("au.com.dius.pact.consumer:junit5")
 }
+
+project.version = "1.0.0"
```

Comment:
Could this be `0.1.0-SNAPSHOT`?

Reply:
Could be whatever. I just wanted round number for the broker frontend.